contingency/contingency-api/src/main/java/com/powsybl/contingency/SidedContingencyElement.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.contingency;
+
+ /**
+ * @author Laurent Issertial {@literal <laurent.issertial at rte-france.com>}
+ */
+public interface SidedContingencyElement extends ContingencyElement {
+
+    String getVoltageLevelId();
+
+    boolean hasVoltageLevelId();
```

Comment:
Is it really necessary? A null voltageLevelId already means there is no voltageLevelId.